src/benchmarker.cpp

```diff
@@ -60,13 +62,16 @@ void Benchmarker::endBenchmark()
   auto timeSinceEpoch = std::chrono::duration_cast<std::chrono::milliseconds>
                 (std::chrono::system_clock::now().time_since_epoch()).count();
 
-  if (printToStdout && showBenchmarks){
+  if (printToStdout){
     std::cout << "[TIME] Ended <"
               << description
               << ">. Elapsed: "
               << std::chrono::duration_cast<std::chrono::milliseconds>(elapsed).count()
-              << " ms. Time since epoch (ms): " << timeSinceEpoch
-              << std::endl;
+              << " ms. ";
+    if (showBenchmarks){
+      std::cout << "Time since epoch (ms): " << timeSinceEpoch;
+    }
+    std::cout << std::endl;
```

Comment:
Please remove the other print statements we had discussed too - refer to our Slack messages.

Reply:
I deleted the ones indicated in #20 and modified `Time since epoch (ms)` to be printed only when `show_benchmarks=true` for the benchmarking script.
Should I delete anything else? Here's a sample of the full output.
https://pastebin.com/7qyWuSCH

Reply:
Looks good, but a few more removals:

```traffic person vec size = 3441953
index path vec size = 222515020
edgesData size = 627199
laneMap size = 1284503552
intersections size = 223328
``` This is printed twice. Only keep one set of them.

Also, please remove:

```Person 0: init 192085 end 91309 time 35570.496094 a 0.557041 b 2.902058 T 0.543303
```

Please remove:

```First time_departure 18000.000000
```

Please remove:

```  >> createLaneMap
```

That's all I'm noticing for now. Thanks!

Reply:
Done! Looks like this now: https://pastebin.com/hc45Gvyh
Is there something else that we should remove or modify?

Reply:
Looks good for now!